systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -391,28 +393,29 @@ public String  getInitContainerImageName(String podName) {
         return JsonPath.parse(clusterOperatorJson).read("$.spec.initContainers[-1].image");
     }
 
-    @BeforeEach
-    void setTestName(TestInfo testInfo) {
-        testName = testInfo.getTestMethod().get().getName();
-    }
-
-    @BeforeAll
-    static void setTestClassName(TestInfo testInfo) {
-        testClass = testInfo.getTestClass().get().getSimpleName();
-    }
-
     protected void createResources() {
         LOGGER.info("Creating resources before the test");
         resources = new Resources(namespacedClient());
     }
 
+    protected static void createClusterOperatorResources() {
+        LOGGER.info("Creating cluster operator resources");
+        clusterOperatorResources = new Resources(namespacedClient());
+    }
+
     protected void deleteResources() throws Exception {
         collectLogs();
         LOGGER.info("Deleting resources after the test");
         resources.deleteResources();
         resources = null;
     }
 
+    protected static void deleteClusterOperatorResources() {
+        LOGGER.info("Deleting cluster operator resources after the test");
+        clusterOperatorResources.deleteResources();
+        clusterOperatorResources = null;
```

Comment:
Necessary?

Reply:
You think that it's redundant because resources will be deleted by GC and pods by openshift during namespace deletion?

Reply:
I think so.